variables.tf

```
@@ -21,6 +21,42 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+variable "cognito_authenticated_role_name" {
+  default     = "dmarc-import-authenticated"
+  description = "The name of the IAM role that grants authenticated access to the Elasticsearch database."
+  type        = string
+}
+
+variable "cognito_identity_pool_name" {
+  default     = "dmarc-import"
+  description = "The name of the Cognito identity pool to use for access to the Elasticsearch database."
+  type        = string
+}
+
+variable "cognito_user_pool_client_name" {
+  default     = "dmarc-import"
+  description = "The name of the Cognito user pool client to use for access to the Elasticsearch database."
+  type        = string
+}
+
+variable "cognito_user_pool_domain" {
+  default     = "dmarc-import"
+  description = "The domain to use for the Cognito endpoint. For custom domains, this is the fully-qualified domain name, such as auth.example.com. For Amazon Cognito prefix domains, this is the prefix alone, such as auth."
```

Just to prevent misreading the prefix as `auth.`.

```suggestion
  description = "The domain to use for the Cognito endpoint. For custom domains, this is the fully-qualified domain name, such as \"auth.example.com\". For Amazon Cognito prefix domains, this is the prefix alone, such as \"auth\"."
```

---

@mcdonnnj Just highlighting the fact that for consistency, I edited your suggestions slightly to also put quotes around the example domain, in addition to the example prefix, as you suggested (see https://github.com/cisagov/cool-dns-dmarc-import/pull/11/commits/f69725646cd667b5d73c4bdd418e6a2391a3dee6).